test/test_bundle_importer.js

```diff
@@ -63,16 +63,35 @@ describe('Bundle Importer', function() {
   });
 
   describe('Public methods', function() {
+
     describe('importBundleYAML', function() {
       it('calls fetchDryRun with yaml', function() {
         var fetch = utils.makeStubMethod(bundleImporter, 'fetchDryRun');
         this._cleanups.push(fetch.reset);
         bundleImporter.importBundleYAML('foo: bar');
         assert.equal(fetch.callCount(), 1);
-        assert.equal(fetch.lastArguments()[0], 'foo: bar');
+        var args = fetch.lastArguments();
+        assert.lengthOf(args, 2);
```

Comment:
can you convert these 'fancy' assert methods to just use 'assert.equal'. Unfortunately the fancy ones give 0 traceback as they get stuck in Chai land.